### PR TITLE
Feature/add mosaic UI

### PR DESCRIPTION
**Pull request by eheinrich**
Add mosaic generation to the compose page. This branch relies on this version of the ocs-component-lib: https://github.com/observatorycontrolsystem/ocs-component-lib/pull/17

### CODE REVIEW

**Review by eheinrich — COMMENTED** [review]
This all looks good to me. Since I opened the PR, I was not able to assign myself as a reviewer so I can't actually click approve, but I approve it 👍

File: src/views/Compose.vue [hunk 1]
```diff
@@ -680,6 +694,26 @@ export default {
...
+      } else if (request.configurations[0].target.type !== 'ICRS') {
+        return false;
+      }
+      // TODO: To release mosaicing, update the line below to remove the is_staff check`
+      return this.$store.state.profile.is_staff && !this.simpleInterface;
+    },
+    extraMosaicRotation: function(configuration) {
+      let rotatorAngle = 0;
+      for (let instConfigIndex in configuration.instrument_configs) {
+        if ('extra_params' in configuration.instrument_configs[instConfigIndex]) {
```
Comment (jnation3406) [inline src/views/Compose.vue:711]: I wonder if we should also check that it has a rotator mode set... if not we should probably ignore a rotator_angle in the extra params

**Review by jnation3406 — APPROVED** [review]
Looks good to me

File: src/views/Compose.vue (hunk 1 above)
Comment (eheinrich) [reply src/views/Compose.vue]: Good idea. I added a check in to make sure the SKY rotator mode is set before applying rotation.

**Pull request merged**